standalone/node/src/service.rs

```diff
@@ -160,6 +161,11 @@ pub fn new_partial(
 		let select_chain = select_chain.clone();
 		let keystore = keystore_container.sync_keystore();
 		let chain_spec = config.chain_spec.cloned_box();
+		let backend = backend.clone();
```

Comment:
Yes, its necessary, to avoid the `backend` to be moved into the `rpc_extensions_builder`.
Futher more, the `backend.clone()` inside the `rpc_extensions_builder` is to avoid the `backend` captured by the closure to be moved out of the closure state.